experimental/transformers-to-ctranslate/args.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+from ctranslate2.converters.converter import Converter
+
+def make_parser():    
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        "--model",
+        required=True,
+        help=(
+            "Name of the pretrained model to download, "
+            "or path to a directory containing the pretrained model."
+        ),
+    )
+    parser.add_argument(
```

Comment:
do not make this an commandline interface, use its default value when converting.